macros/upload_invocations.sql

```diff
@@ -56,10 +57,17 @@
             {% for dbt_var in var('dbt_vars') %}
                 {% do dbt_vars_dict.update({dbt_var: var(dbt_var)}) %}
             {% endfor %}
-            '{{ tojson(dbt_vars_dict) }}' {# dbt_vars #}
+            '{{ tojson(dbt_vars_dict) }}', {# dbt_vars #}
         {% else %}
-            null {# dbt_vars #}
+            null, {# dbt_vars #}
         {% endif %}
+        
+        {% set invocation_args = {} %}
+        {% for arg, value in invocation_args_dict.items() %}
+            {% do invocation_args.update({arg: value}) %}
+        {% endfor %}
+        '{{ tojson(invocation_args) }}' {# invocation_args #}
```

Comment:
```suggestion
        {% set invocation_args = {} %}
        {% for arg, value in invocation_args_dict.items() %}
            {% do invocation_args.update({arg: value}) %}
        {% endfor %}
        '{{ tojson(invocation_args) }}' {# invocation_args #}
```
```suggestion
        '{{ tojson(invocation_args_dict) }}' {# invocation_args #}
```

Is that the same?

Reply:
Yes it is. Have committed your suggestion so there is less unnecessary code. Thanks for the spot!